### ISSUE TITLE
BreadCrumb: items are not announced correctly using NVDA. 

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#2035]
### Bug Report

- __Package version(s)__: 2.20.2
- __Browser and OS versions__: All browsers, windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: VSOnline 

#### Describe the issue:

Breadcrumb items are not announced correctly when focused using NVDA.
In edge and firefox it reads characters of the item text as we move focus from one to next.
In chrome, it reads previously focused breadcrumb item and currently focused item both.

#### Actual behavior:

Breadcrumb items are not announced correctly when focused using NVDA.
In edge and firefox it reads characters of the item text as we move focus from one to next.
In chrome, it reads previously focused breadcrumb item and currently focused item both.

#### Expected behavior:

It should be announced properly.

### If applicable, please provide a codepen repro:

https://codepen.io/anon/pen/bRWyRB

**Comment by b-priyank** [comment]
@atneik Is there any update on this?
